### PR TITLE
finished project, all topics done

### PR DESCRIPTION
**Pull request by mgmediaweb**
The project is done, I follow all the requirements and work fine, with no linter and webhint error

### CODE REVIEW

File: Assets/css/style.css [hunk 1]
```diff
@@ -1,3 +1,194 @@
...
+    width: 100%;
+  }
+
+  main img {
+    bottom: -50px;
+    position: absolute;
+    height: 450px;
+    right: -150px;
+    width: auto;
+  }
```
Comment (CollinsTatang) [inline Assets/css/style.css:149]: - [ ] Please add this image as a background image it is actually floating over the text.
float

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @mgmediaweb , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:

## Required Changes ♻️

- [ ] Please do your best to improve on your pull request title and summary by using the [guidelines](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -2,12 +2,51 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;700&display=swap" rel="stylesheet">    
     <link rel="stylesheet" href="Assets/css/style.css" />
 </head>
 <body>
-    <h1>Gonzalo's Portfolio</h1>    
+    <header>
+        <div class="container header-flex">
+            <h1>Gonzalo A. Medina</h1>
```
Comment (mahbubul14) [inline index.html:21]: - [ ] As the previous reviewer suggested, Please use `a tag` instead of  `h1 tag` here. :+1:

File: index.html [hunk 3]
```diff
@@ -2,12 +2,51 @@
...
+        </div>        
+    </header>
+    <main>
+        <div class="main-presentation">
+            <div class="social-icons">
+                <a href="https://github.com/mgmediaweb" target="new"><i class="fab fa-github social-icon"></i></a>
+                <a href="https://www.linkedin.com/in/gonzalo-medina-g/" target="new"><i class="fab fa-linkedin-in social-icon"></i></a>
+                <a href="https://twitter.com/GonzoMedinaDev" target="new"><i class="fab fa-twitter social-icon"></i></a>
+                <a href="https://www.facebook.com/Gonzobike/" target="new"><i class="fab fa-facebook social-icon"></i></a>
+            </div>
```
Comment (mahbubul14) [inline index.html:42]: As the previous reviewer suggested, please wrap the list of icons in `ul tag` to group them and each icon should be in a `li tag` and has a link with a` tag`. To get an idea you can check the following example.

```
<ul class="social-icons">
  <li><a href="https://github.com/mgmediaweb" target="new"><i class="fab fa-github social-icon"></i></a></li>
</ul>`
```

File: Assets/css/style.css [hunk 4]
```diff
@@ -1,3 +1,194 @@
...
+
+  nav {
+    display: none;
+  }
+
+  .main-presentation {
+    box-sizing: border-box;
+    display: flex;
+
+    /* flex: 1 1; */
```
Comment (mahbubul14) [inline Assets/css/style.css:168]: - [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly remove the inline comments from your code. :+1:

File: Assets/css/style.css (hunk 1 above)
Comment (mahbubul14) [reply Assets/css/style.css:149]: - [ ] As the previous reviewer suggested, Kindly add this image as a background image it is actually floating over the text. You can check [this link](https://www.w3schools.com/cssref/pr_background-image.asp#:~:text=By%20default%2C%20a%20background%2Dimage,if%20the%20image%20is%20unavailable.) to check how to set a proper background image.

m1

File: index.html [hunk 5]
```diff
@@ -2,12 +2,51 @@
...
+                    <li><a href="#"><i class="far fa-envelope"></i></a></li>  
+                </ul>
+            </nav>
+            <div class="movil-menu">
+                <a href="#"><i class="fas fa-bars"></i></a>
+            </div>
+        </div>        
+    </header>
+    <main>
+        <div class="main-presentation">
```
Comment (mahbubul14) [inline index.html:36]: - [ ] As the previous reviewer suggested, You can use `section tag` after the `main tag` for better HTML semantics.

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @mgmediaweb ,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Clean code.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter should be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**